include/jemalloc/internal/arena_inlines_b.h

```diff
@@ -55,10 +55,14 @@ arena_prof_info_get(tsd_t *tsd, const void *ptr, alloc_ctx_t *alloc_ctx,
 
```

Comment:
The !unlikely above this line is rather confusing. Is it meant to be unlikely(!(...)) ?Let’s fix it either here or in another PR.

Reply:
Good catch! This is from an earlier commit, so I'm putting the fix as a separate commit.